site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -9,7 +9,73 @@ docs: "DOCS-1419"
 
 This topic describes possible issues users might encounter when using NGINX Gateway Fabric. When possible, suggested workarounds are provided.
 
-### NGINX fails to reload
+### **General Troubleshooting**
```

Comment:
do we need to apply this technique of placing header sections in `**` across all docs? 
It's better to be consistent

Reply:
No, I just thought that the headers in this particular doc might run together too much, so I wanted to create some distinction. I'll actually revert this and use a different header type for the nested ones.